packages/rke2-canal/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-packageVersion: 00
+packageVersion: 01
```

Comment:
Could you check the file termination? Somehow there is this character that github doesn't recognize

Reply:
now it's ok